test/utilities/git.jl

```diff
@@ -362,3 +362,43 @@ end
         end
     end
 end
+
+@testset "remote_exists" begin
+    mktempdir() do f
+        cd(f) do
+            run(`git init`)
+            run(`git remote add origin foo-url`)
+            run(`git remote add upstream bar-url`)
+
+            @testset "remote exists" begin
+                @test CompatHelper.remote_exists("origin")
+                @test CompatHelper.remote_exists("upstream")
+            end
+            @testset "remote does not exist" begin
+                @test !CompatHelper.remote_exists("nonexistent")
+            end
+        end
+    end
+end
+
+@testset "git_remote_add_or_seturl" begin
+    mktempdir() do f
+        cd(f) do
+            run(`git init`)
+            run(`git remote add origin foo-url`)
+
+            @testset "set existing remote" begin
+                url = "foo-url"
+                CompatHelper.git_remote_add_or_seturl("origin", "foo-url")
+                output = strip(read(`git remote get-url origin`, String))
+                @test output == url
```

Comment:
Can you now do `git_remote_add_or_seturl()` with a different URL value, and then confirm that the new URL was set correctly?

Reply:
Oops, that's silly of me.

Reply:
Sorry, I should clarify. You first have a test that sets the URL to `bar-url` and checks that it was set correctly. Now, immediately following that test, can you add a second test that changes the URL to `bar-url2` and checks that the change went through correctly?

Reply:
Can do!